composer.json

```diff
@@ -7,6 +7,7 @@
     "require": {
         "php": ">=7.2.0",
         "ext-json": "*",
+        "psr/log": "^1 || ^2 || ^3",
```

Comment:
why 2 different psr/log dep versions ?

Reply:
That was my suggestion.

The set of tests the bundle already has relies on a psr/log's 1.1 test logger, which is weird, but OK, let's allow it for now, not to broaden the scope of the PR.

Therefore, the dev dependencies MUST rely on that particular version, while the bundle itself should NOT restrict its users from taking advantage of versions 2 and 3.

Reply:
We have a problem with the dependency in the code, when we change from version 1 to version 2 or 3, some files do not exist, now it is necessary to include the version only for **require-dev** and include another version in **require**.

This is not the best solution but it is necessary so as not to change many tests.

Reply:
I mean why not `"^1.1.0 || ^2 || ^3"` ?

Reply:
The no-dev doesn't depend on 1.1, but sure, I don't see why not.